src/templates/Header/Header.module.scss

```diff
@@ -0,0 +1,8 @@
+.graylinks {
```

Comment:
for class names, use [BEM](https://getbem.com/introduction/) naming pattern. For example `gray-links`.

Why `greylinks`? These links are not gray

Reply:
resolved in f29528d